src/Stratis.StraxD/Program.cs

```diff
@@ -52,11 +52,7 @@ public static async Task Main(string[] args)
                     .UseSmartContractPosPowMining()
                     .UseApi()
                     .AddRPC()
-                    .AddSmartContracts(options =>
-                    {
-                        options.UseReflectionExecutor();
-                        options.UsePoSWhitelistedContracts();
-                    })
+                    .AddSystemContracts()
```

Comment:
This is probably not the only daemon that requires updating. For instance some fedpeg daemons would be creating mainchain daemons when called with the -mainchain argument.